stringslab.js

```diff
@@ -0,0 +1,158 @@
+// # String Exercises
```

Comment:
I saw some correct use of string methods as well as bad practice of hard coding. Please revisit comments below and try to solve some problems again. Keep practicing and keep solving bonus questions.